js/weather/weather.js

```diff
@@ -0,0 +1,45 @@
+import { config } from './apikey.js';
+
+
+function getLocation(){
+  if(navigator.geolocation){
+    navigator.geolocation.getCurrentPosition(showPosition);
+  }else{
+    console.log('Geolocation not available');
```

Comment:
@zaynaib Would this be used for future testing?

Reply:
I made it into an alert.